Community/acl_inventory/templates/acl_inventory_page.html

```diff
@@ -0,0 +1,25 @@
+<!-- Copyright 2018 BlueCat Networks. All rights reserved. -->
+
+{% extends "base.html" %}
```

Comment:
this really isn't an issue but, if I saw that you try to extend the width of the to 120% in the CSS,
if you change "base.html" to "base-no-box.html" it will make the page much wider like our admin workflows (general/mail).

Reply:
True, I do that sometimes as well.